test: 거래내역 컨트롤러 테스트

nojamcode opened the pull request:
## 작업 분류

- [ ] 버그 수정
- [x] 신규 기능 추가
- [ ] 리팩터링
- [ ] 테스트

## 작업 개요
- 입금, 출금 컨트롤러 테스트 추가
- 입금, 출금 유닛테스트 추가


resolves: #59

Commit:
test:입금,출금 컨트롤러 테스트 추가

Commit:
test:입금,출금 컨트롤러 테스트 추가

chinsanchung reviewed (commented):
잠시만요. 컨트롤러 테스트에 목록하고 조회도 추가해서 업로드할게요.

chinsanchung committed:
Merge branch 'test/transaction-controller' of https://github.com/wanted-wecode-subjects/eight-percent-subject into test/transaction-controller

chinsanchung committed:
test: 목록 조회, 목록 상세 보기 테스트 코드 작성

- 입금 목록과 출금 목록을 테스트하는 코드를 작성했습니다.
- 특정 목록의 상세 보기를 테스트하는 코드를 작성했습니다.

resolves: #16
Co-Authored-By: BangleCoder <47234375+BangleCoding@users.noreply.github.com>

lhj0621 committed:
Merge branch 'develop' into test/transaction-controller

lhj0621 committed:
test: mockListQuery 변경

account_ld -> acc_num

resolves: #59
